### ISSUE TITLE
feat(`chisel`): add in-line assembly { .. } mode

### ISSUE THREAD

**Issue by kadenzipfel** [foundry-rs/foundry#4963]
### Component

Chisel

### Describe the feature you would like

I most commonly find myself using chisel to identify the result of assembly logic, but find it requires an extra step to work with, i.e. if I want to get the result of something simple, like `not(0)`, I have to do something like:

```
> uint256 a
> assembly { a := not(0) }
> a
```

Instead, it would be super useful if I could somehow indicate that I'm passing assembly logic directly and to output the value of that logic, effectively doing the above behind the scenes

**Comment by mds1** [comment]
Agreed this would be really useful. Not sure how feasible it is, but maybe by default the last statement in the assembly block has the result printed by default? cc @clabby
